pkg/hive/hive_test.go

```diff
@@ -232,6 +232,72 @@ func TestHiveCIDRSlice(t *testing.T) {
 	require.Equal(t, cidr.MustParseCIDR("2001:db8::/64"), cfg.Foo[1], "Config.Foo not set correctly")
 }
 
+type StringSliceConfig struct {
+	SpacesFlag, CommasFlag []string
+	SpacesMap, CommasMap   []string
+
+	Mixed []string
```

Comment:
It could make sense adding another `StringFlag string` entry, and assert that it is neither split on commas not spaces.

Reply:
You mean to assert that a `StringFlag string` that's marked as `flags.String("string-flag", ...)` isn't mangled somehow if it contains commas or spaces?

Reply:
Yep, that was the idea. Reading that again, though, it doesn't look quite possible to incorrectly mangle it, as the types would not match at all.

Reply:
Added. Check if that's what you meant :)

Reply:
Yep, thanks!